script/lib/download-standalone.sh

```diff
@@ -7,7 +9,7 @@ require_binary curl
 require_binary unzip
 require_env_var STANDALONE_VERSION
 
-BASEURL=https://github.com/pact-foundation/pact-ruby-standalone/releases/download
+BASEURL=https://github.com/you54f/pact-ruby-standalone/releases/download
```

Comment:
I don't think this is meant to be in this PR

Reply:
currently we are rocking my fork for pact ruby standalone until this pr is merged

https://github.com/pact-foundation/pact-ruby-standalone/pull/105

allowed me to test the end deliverable across multiple projects prior to releasing the pact ruby standalone officially.